CorsixTH/Lua/room.lua

```diff
@@ -722,6 +722,20 @@ function Room:roomFinished()
   self:calculateHappinessFactor()
 end
 
+--! Check if a room is actively required by patients
+--! A room is deemed in demand should any patient be in a state of using, or
+--! wanting to use the room.
+--!return (boolean) true if the room is currently needed
+function Room:isRoomInDemand()
+  local door = self.door
+  if self:getPatientCount() > 0 and not self:getPatient():isLeaving() then
```

Comment:
I think this is wrong for the ward if patient-centered > 1.

Reply:
Good point. Likely this was broken before. Will also check the changes that let the nurse leave ward before fixing but I don't think they included being able to find new work.

Reply:
I closed this by accident